questionpy/_attempt.py

```diff
@@ -237,7 +238,10 @@ def _compute_final_score(self) -> float:
 
     @cached_property
     def jinja2(self) -> jinja2.Environment:
-        return create_jinja2_environment(self, self.question)
+        # Get the caller module.
+        frame_info = inspect.stack(context=0)[2]  # Second entry, because of the decorator.
```

Comment:
Warum nicht nach dem Modul gehen, in dem der Attempt implementiert wurde?
Hast du überprüft, welches Modul man mit `self.__module__` erhält?

Reply:
`__module__` kann überschrieben werden: auf irgendeinen Wert oder auch zu `namespace.shortname` eines anderen Pakets. Ich wüsste allerdings nicht, wie man das böswillig ausnutzen könnte und habe es jetzt zu `attempt.__module__` umgeändert.